### PR TITLE
[PMD] ERROR: Avoid unused private fields

### PR DESCRIPTION
**Pull request by david-guzman**

### CODE REVIEW

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java [hunk 1]
```diff
@@ -42,11 +42,11 @@
  */
 public class EidpLoginModule implements LoginModule {
 
-  private Subject subject;
+//  private Subject subject;
```
Comment (sputnikci) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: \[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 2, indentation should be the same level as line 46\.

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java [hunk 2]
```diff
@@ -42,11 +42,11 @@
...
   private CallbackHandler callbackHandler;
-  private Map<String, ?> sharedState;
-  private Map<String, ?> options;
-  private boolean debug = false;
+//  private Map<String, ?> sharedState;
+//  private Map<String, ?> options;
+//  private boolean debug = false;
```
Comment (sputnikci) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: \[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 2, indentation should be the same level as line 51\.

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java [hunk 3]
```diff
@@ -70,11 +70,11 @@ private void setUserService() {
 
   @Override
   public void initialize(Subject subject, CallbackHandler callbackHandler, Map<String, ?> sharedState, Map<String, ?> options) {
-    this.subject = subject;
+//    this.subject = subject;
```
Comment (sputnikci) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: \[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 74\.

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java [hunk 4]
```diff
@@ -70,11 +70,11 @@ private void setUserService() {
...
     this.callbackHandler = callbackHandler;
-    this.sharedState = sharedState;
-    this.options = options;
-    this.debug = "true".equalsIgnoreCase((String)options.get("debug"));
+//    this.sharedState = sharedState;
+//    this.options = options;
+//    this.debug = "true".equalsIgnoreCase((String)options.get("debug"));
```
Comment (sputnikci) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: \[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 78\.

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java (hunk 2 above)
Comment (david-guzman) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: Assigned #14

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java (hunk 3 above)
Comment (david-guzman) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: Assigned #14

File: eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java (hunk 4 above)
Comment (david-guzman) [inline eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java]: Assigned #14

**Comment by david-guzman** [comment]
Closes #14

**Pull request merged**